### PR TITLE
Update portStatIds for cisco

### PR DESCRIPTION
**Pull request by zhixzhu**
**What I did**
Update portStatIds for cisco

**Why I did it**
test_pfcwd_status.py failed with error 'E Failed: FLEX DB does not properly reflect Pfcwd status', script expect 3 entries for each port, but it only got 2 entries. The port level entry is missing.

**How I verified it**
Verified it on testbed.
Before the fix:
```
cisco@sonic:~$ sudo pfcwd start --action drop --restoration-time 400 Ethernet128 300
cisco@sonic:~$ sonic-db-cli -n asic1 FLEX_COUNTER_DB keys *FLEX_COUNTER_TABLE:PFC_WD:oid:0x1500000000006*
FLEX_COUNTER_TABLE:PFC_WD:oid:0x15000000000064
FLEX_COUNTER_TABLE:PFC_WD:oid:0x15000000000065
cisco@sonic:~$ sonic-db-cli -n asic1 FLEX_COUNTER_DB keys *0x1000000000004*
FLEX_COUNTER_TABLE:PORT_STAT_COUNTER:oid:0x1000000000004
FLEX_COUNTER_TABLE:PORT_BUFFER_DROP_STAT:oid:0x1000000000004
```

After the fix:
```
cisco@sonic:~$ sudo pfcwd start --action drop --restoration-time 400 Ethernet128 300
cisco@sonic:~$ sonic-db-cli -n asic1 FLEX_COUNTER_DB keys *0x1000000000004*
FLEX_COUNTER_TABLE:PORT_STAT_COUNTER:oid:0x1000000000004
FLEX_COUNTER_TABLE:PORT_BUFFER_DROP_STAT:oid:0x1000000000004
FLEX_COUNTER_TABLE:PFC_WD:oid:0x1000000000004    //This is new entry after fix
cisco@sonic:~$ sonic-db-cli -n asic1 FLEX_COUNTER_DB keys *FLEX_COUNTER_TABLE:PFC_WD:oid:0x1500000000006*
FLEX_COUNTER_TABLE:PFC_WD:oid:0x15000000000064
FLEX_COUNTER_TABLE:PFC_WD:oid:0x15000000000065

127.0.0.1:6379[5]> hgetall FLEX_COUNTER_TABLE:PFC_WD:oid:0x1000000000004
1) "PORT_COUNTER_ID_LIST"
2) "SAI_PORT_STAT_PFC_3_RX_PKTS,SAI_PORT_STAT_PFC_4_RX_PKTS,SAI_PORT_STAT_PFC_3_TX_PKTS,SAI_PORT_STAT_PFC_4_TX_PKTS"
```
**Details if related**

### CODE REVIEW

**Comment by zhixzhu** [comment]
@prsunny There is 1 workflow requires approval from a maintainer, could you help approve it?

**Review by kevinskwang — APPROVED** [review]

**Comment by zhixzhu** [comment]
@XuChen-MSFT There is 1 workflow requires approval from a maintainer, could you help approve it?

**Review by prsunny — APPROVED** [review]

**Pull request merged**
